sky/server/server.py

```diff
@@ -1904,6 +1904,8 @@ async def kubernetes_pod_ssh_proxy(
         client_version = {client_version}')
 
     # Run core.status in another thread to avoid blocking the event loop.
+    # TODO(aylei): core.status() will be called with server user, which has
+    # permission to all workspaces, this will break workspace isolation.
```

Comment:
```suggestion
    # TODO(aylei): core.status() will be called with server user, which has
    # permission to all workspaces, this will break workspace isolation.
    # It is ok for now, as users with limited access will not get the ssh config
    # for the clusters in non-accessible workspaces.
```